src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { SyPersonioJob } from '../../../types';
+
+interface CareerDetailsStructuredDataProps {
+  position: SyPersonioJob;
+}
+
+const convertScheduleToEmploymentType = (schedule: string): string => {
+  return schedule.toUpperCase().replace('-', '_');
+};
+
+const concatDescription = (position: SyPersonioJob): string => {
+  return (
+    position.short +
+    '<br><br>' +
+    position.sections
+      .map((section) => `${section.headline} <br> ${section.descriptionHtml}`)
+      .join('<br><br>')
+  );
+};
+
+export const CareerDetailsStructuredData = ({
+  position,
+}: CareerDetailsStructuredDataProps) => {
+  const employmentType = convertScheduleToEmploymentType(position.schedule);
+
+  const data = {
+    '@context': 'https://schema.org/',
```

Comment:
Please add a bunch of comments about what this structured data is and what some of the hardcoded values mean. Usually, a developer doesn't know anything about it, which makes it very important to add enough comments to make it clear.

Reply:
I have included the the typings and an overalll comment as I found it difficult to understand too.